src/main/java/com/atlantbhinternship/auctionapp/responses/LoginResponse.java

```diff
@@ -0,0 +1,18 @@
+package com.atlantbhinternship.auctionapp.responses;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@Data
```

Comment:
https://projectlombok.org/features/Data

This annotation will create some duplicate code e.g. equals and hash function which you already declare above.

I advise to remove all unnecessary lombok annotations from all files.